crates/sozo/src/commands/options/account.rs

```diff
@@ -33,7 +33,6 @@ pub struct AccountOptions {
 
     #[arg(long = "password", env = DOJO_KEYSTORE_PASSWORD_ENV_VAR)]
     #[arg(value_name = "PASSWORD")]
-    #[arg(requires = "keystore_path")]
```

Comment:
removed this require so that when passing `keystore_password` we can still get `keystore_path` from environment metadata